### ISSUE TITLE
rotated pages

### ISSUE THREAD

**Issue by LiviuSebastian** [barteksc/AndroidPdfViewer#585]
Hello! Great library. It helps me a lot.

It is a great thing that it is correctly displaying documents that contain both portrait and landscape pages.

However, in my application I need to know which page is rotated and by which angle. Somehow your library "knows" which page is rotated because as I just said it is correctly displaying documents that contain both portrait and landscape pages.

Is there a way I can use this library to find the rotation angle for each page of a document?

**Comment by 1stmetro** [comment]
I dont believe the code is aware of the rotation of a page, it just knows when to resize to fill the space so i think you are going to struggle with angles.  Remember what you see on the page is an image the codes doesnt know if the image is landscape or portrait it just knows to fill the page.
